crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -100,6 +100,19 @@ def knife_files
           filecontent_replace(file, "nova_dashboard", "horizon")
         end
       end
+
+      # find admin node and update target_platform
+      nodes_path = knife_path.join("nodes")
+      nodes_path.children.each do |file|
+        save = false
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `save`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)